src/plugins/auto/src/plugin.cpp

```diff
@@ -182,10 +158,12 @@ std::vector<DeviceInformation> Plugin::parse_meta_devices(const std::string& pri
         try {
             auto device_id = get_core()->get_property(device_name, ov::device::id);
             return device_id;
+            OPENVINO_SUPPRESS_DEPRECATED_START
         } catch (const InferenceEngine::Exception&) {
```

Comment:
Maybe we need fix `exception catching type` issue to remove `InferenceEngine::Exception` if any place need catch this exception type.